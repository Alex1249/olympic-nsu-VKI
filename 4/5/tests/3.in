9 18
-934 -823 -364 38 513 757 -488 -133 114
1 55
0 212
0 88
1 207
0 59
1 113
0 243
0 213
0 112
1 54
1 57
0 42
1 188
0 141
1 66
1 187
0 230
0 194
