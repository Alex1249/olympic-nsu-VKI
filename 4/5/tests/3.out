-934 -2097207 -492 16422 513 765 -536874472 -33558661 114 
